subsys/bluetooth/audio/mcc.c

```diff
@@ -1193,21 +1180,302 @@ static uint8_t discover_include_func(struct bt_conn *conn,
 	}
 	return BT_GATT_ITER_STOP;
 }
+
+/* Start discovery of included services */
+static void discover_included(struct bt_conn *conn)
+{
+	int err;
+
+	discover_params.start_handle = cur_mcs_inst->start_handle;
```

Comment:
Could it be an issue that`discover_params` are global (not per mcs instance?). What happens when connected to 2 devices in coordinated set?